doc/source/guide/spatial-partitioning.ipynb

```diff
@@ -0,0 +1,383 @@
+    "Future work will examine various methods for better representating complex geometries.\n",
+    "\n",
+    "The Geohash, not to be confused with [geohashing](https://geohashing.site/geohashing/Main_Page) is a hierarchical spatial data structure, which subdivides space into buckets of grid shape.\n",
+    "Whilst the Geohash does allow binary searchers or spatial indexing, like rtree or quadtree, (a topic for later discussion & an enhancement for Dask-GeoPandas), Geohash is limited to encoding latitude and longitude coordinates.\n",
+    "The Geohash can be represented either as text string or integers, where the longer the string/integer is, the more precise the grid shape \n",
+    "will be.\n",
+    "\n",
+    "Below we compare the different sorting methods using the default parameters.\n",
+    "All 3 sorting methods reduce the degree of spatial overlap but this stil remains high and could be further improved by controlling specific parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0ad73df8-8264-4cd8-aa88-185d82ac5305",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hilbert = d_gdf.spatial_shuffle(by=\"hilbert\")\n",
+    "morton = d_gdf.spatial_shuffle(by=\"morton\")\n",
+    "geohash = d_gdf.spatial_shuffle(by=\"geohash\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1fcac318-06de-4144-9d34-b510bddda81e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(nrows=1,ncols=3, figsize=(25,12))\n",
+    "ax1, ax2, ax3 = axes.flatten()\n",
+    "\n",
+    "for ax in axes:\n",
+    "    us_cont.plot(ax=ax)\n",
+    "\n",
+    "hilbert.spatial_partitions.plot(ax=ax1, cmap=\"tab20\", alpha=0.5)\n",
+    "morton.spatial_partitions.plot(ax=ax2, cmap=\"tab20\", alpha=0.5)\n",
+    "geohash.spatial_partitions.plot(ax=ax3, cmap=\"tab20\", alpha=0.5)\n",
+    "\n",
+    "[axi.set_axis_off() for axi in axes.ravel()]\n",
+    "\n",
+    "ax1.set_title(\"Hilbert\", size=16)\n",
+    "ax2.set_title(\"Morton\", size=16)\n",
+    "ax3.set_title(\"Geohash\", size=16)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e34f773c-0eec-4a78-bc78-66dd6c4ebd22",
+   "metadata": {},
+   "source": [
+    "## npartitions\n",
+    "The first parameter is the `npartitions` or number of partitions.\n",
+    "The default npartitions is the same number of partitions as the original Dask GeoDataFrame.\n",
+    "Whilst increasing the number of partitions will reduce the degree of spatial overlap between partitions, this is also a more expensive operation.\n",
+    "Knowing how many partitions to set is tricky because there is no single answer.\n",
+    "This depends not only on your data structure but also what you plan to do with your data.\n",
+    "Overall, there is a trade off between the up front and geometric operations cost when deciding how to spatially shuffle a Dask GeoDataFrame."
```

Comment:
Can you rephrase this? I am not sure I am following what you're trying to say.

Reply:
Something like _Breaking your DataFrame into "too many" partitions results in [the scheduler incurring more overhead deciding where to compute each task](https://docs.dask.org/en/stable/dataframe-best-practices.html#repartition-to-reduce-overhead]._ ?